communities/microgalaxy/lab/sections/1_data_import_and_preparation.yml

```diff
@@ -62,15 +62,15 @@ tabs:
         description_md: >
             SPAdes implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.
             <ul>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_biosyntheticspades">spades_biosyntheticspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_coronaspades">spades_coronaspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_metaplasmidspades">spades_metaplasmidspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fmetaspades">metaspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_metaviralspades">spades_metaviralspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_plasmidspades">spades_plasmidspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Frnaspades">rnaspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades_rnaviralspades">spades_rnaviralspades</a></li>
-              <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades%2Fspades">spades</a></li>
+              <li><a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/spades_biosyntheticspades/spades_biosyntheticspades/3.15.5+galaxy2">spades_biosyntheticspades</a></li>
```

Comment:
Could you remove all version number here? in case a server has a different version

Reply:
I will also add the short description to the interactive tools